packages/ui/src/components/table-v2.tsx

```diff
@@ -0,0 +1,95 @@
+import { forwardRef, HTMLAttributes, ReactNode, TdHTMLAttributes, ThHTMLAttributes } from 'react'
+
+import { cn } from '@utils/cn'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+const tableVariants = cva('cn-table-v2', {
+  variants: {
+    variant: {
+      default: 'cn-table-v2-default',
+      relaxed: 'cn-table-v2-relaxed',
+      compact: 'cn-table-v2-compact'
+    }
+  },
+  defaultVariants: {
+    variant: 'default'
+  }
+})
+
+export interface TableRootV2Props extends HTMLAttributes<HTMLTableElement>, VariantProps<typeof tableVariants> {
+  disableXScroll?: boolean
+  tableClassName?: string
+}
+
+const TableRoot = forwardRef<HTMLTableElement, TableRootV2Props>(
+  ({ variant, disableXScroll, className, tableClassName, ...props }, ref) => (
+    <div className={cn('cn-table-v2-container', tableVariants({ variant }), className)}>
```

Comment:
why do we need this div wrapper? can we not apply the stlying directly on the `table` element?